api/models/Tank.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const mongoose= require("mongoose");
+
+//tank schema
+
```

Comment:
**Add a TODO here!**

It's better to use [GeoJSON](https://mongoosejs.com/docs/geojson.html) for the location. In the long run, it seems good to me as just storing the location as string won't serve the purpose of pointing the exact location on the map. What say @ArpitKotecha?

Reply:
Yeah that will be much better.